zuul.d/project.yaml

```diff
@@ -16,6 +16,7 @@
         - ansible-network-vyos-appliance
     gate:
       jobs:
+        - ansible-cloud-vcenter-appliance
```

Comment:
This needs to be removed, we flipped the job non-voting because of timeout failure we seen from zuul-executor

Reply:
oh, ok let's me fix that

Reply:
Done